admin/docs/customizing_main_navigation.md

```diff
@@ -0,0 +1,48 @@
+# Customizing the main navigation
+
+You are allowed to add your custom links to the main navigation. To do so, you can access `SolidusAdmin::Config.main_nav` in an initializer:
+
+```ruby
+# config/initializers/solidus_admin.rb
+SolidusAdmin::Config.configure do |config|
+  config.main_nav do |main_nav|
+    main_nav.add(
+      key: :my_custom_link,
+      route: :my_custom_link_path,
+      icon: "solidus_admin/my_custom_link.svg",
+      position: 80
+    )
+  end
+end
+```
+
+- The key you provide will be used to translate the link's label under the
+`solidus_admin.main_nav.#{key}` key. 
+- Icon needs to be a path to a SVG file accessible from the asset pipeline.
```

Comment:
I think we need to provide and document the full icon set somehow (let's wait for the answer to the related Elia's comment) because it's unrealistic that people will build their own icon if they want to add a new item to the menu.

Reply:
Done!